### PR TITLE
Enable UI Tests to run in Docker container

### PR DESCRIPTION
**Pull request by NKannan14**
Updated Dockerfile to install chromium and chrome driver to enable running tests in a docker container
Modified `run-tests.sh` to include the UI tests
Introduced isDocker variable to determine if the test are running in a Docker environment and conditionally load webdriver service based on the environment. Only `chromeDriver` will be loaded when running tests in Docker
Added new script in `package.json` for running tests in Docker with the IS_DOCKER=true environment variable

### CODE REVIEW

File: integration_tests/run-tests.sh [hunk 1]
```diff
@@ -19,6 +19,12 @@ TESTS_EXIT_CODE=$?
 
 cp reports/testReport.xml $TEST_REPORT_ABSOLUTE_DIR/junit.xml
 
-if [ $TESTS_EXIT_CODE -ne 0 ]; then
+npm run test:ui
+
+TESTS_EXIT_CODE=$?
```
Comment (phinterseer) [inline integration_tests/run-tests.sh:24]: You're using the same variable twice. You need two different ones of course.

File: integration_tests/run-tests.sh [hunk 2]
```diff
@@ -19,6 +19,12 @@ TESTS_EXIT_CODE=$?
 
 cp reports/testReport.xml $TEST_REPORT_ABSOLUTE_DIR/junit.xml
 
-if [ $TESTS_EXIT_CODE -ne 0 ]; then
+npm run test:ui
```
Comment (phinterseer) [inline integration_tests/run-tests.sh:22]: You might want to abort before this if the int-tests fail. But that's ultimately for you to decide if you think it's worthwhile running the ui-tests with failing int-tests.

File: integration_tests/run-tests.sh (hunk 1 above)
Comment (NKannan14) [reply integration_tests/run-tests.sh]: yes thanks fixed it

File: integration_tests/run-tests.sh (hunk 2 above)
Comment (NKannan14) [reply integration_tests/run-tests.sh:22]: Its worth to run the ui-tests even with failing int-tests as required UI data is directly uploaded to btm_extract_data folder

**Review by phinterseer — APPROVED** [review]
Approved assuming local testing was successful.

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**